validate/src/request.rs

```diff
@@ -117,6 +117,38 @@ pub const USERNAME_LIMIT_MAX: usize = 32;
 /// Minimum length of a username.
 pub const USERNAME_LIMIT_MIN: usize = 2;
 
+/// Maximum length of a webhook username.
+pub const WEBHOOK_USERNAME_LIMIT_MAX: usize = 80;
+
+/// Minimum length of a webhook username.
+pub const WEBHOOK_USERNAME_LIMIT_MIN: usize = 2;
+
+/// String value of an at sign.
+const AT_SIGN: &str = "@";
+
+/// String value of a backslash and three grave symbols.
+const BACKSLASH_THREE_GRAVES: &str = r#"\```"#;
+
+/// String value of 'clyde'.
+const CLYDE: &str = "clyde";
+
+/// String value of a colon.
+const COLON: &str = ":";
+
+/// String value of 'discord'.
+const DISCORD: &str = "discord";
```

Comment:
Should be `discordtag` cf. https://github.com/discord/discord-api-docs/pull/4625

Reply:
It is `discord`. `discordtag` has been changed to `discord` in this PR.
But also `discord` is currently not on the banned list for webhook names.

Reply:
ah, it follows the nickname standards, not usernames.